shell/src/test/java/alluxio/cli/fsadmin/command/ConfigurationCommandTest.java

```diff
@@ -50,13 +50,12 @@ public void configuration() throws IOException {
       String output = new String(outputStream.toByteArray(), StandardCharsets.UTF_8);
       // CHECKSTYLE.OFF: LineLengthExceed - Much more readable
       List<String> expectedOutput = Arrays.asList("Alluxio configuration information: ",
-          "Property                          Value                  Source                                               ",
-          "alluxio.master.port               19998                  DEFAULT                                              ",
-          "alluxio.master.web.port           19999                  DEFAULT                                              ",
-          "alluxio.master.hostname           localhost              SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)",
-          "alluxio.underfs.address           hdfs://localhost:9000  SITE_PROPERTY (/alluxio/conf/alluxio-site.properties)",
-          "alluxio.logger.type               MASTER_LOGGER          SYSTEM_PROPERTY                                      ",
-          "alluxio.master.audit.logger.type  MASTER_AUDIT_LOGGER    SYSTEM_PROPERTY                                      ");
+          "alluxio.master.port (value = 19998, source = DEFAULT)",
```

Comment:
we could make this shorter without losing information with
```
alluxio.master.port = 19998 (source = DEFAULT)
```

Reply:
Great idea! Changed to this format to make information clearer.